bin/app-entrypoint.sh

```diff
@@ -11,5 +11,11 @@ echo "=== Preparing application for ${APP_ENV} environment"
 # finish install scripts
 composer run post-install-cmd
 
+DATA_VOLUME_HOST_PATH="${DATA_VOLUME_HOST_PATH:-}"
```

Comment:
Daemon bude `DATA_VOLUME_HOST_PATH` nastavovat vzdy, ale pro jistotu to tu nechavam optional pro kompatibilitu.